platform/linux-generic/Makefile.am

```diff
@@ -257,16 +257,9 @@ __LIB__libodp_linux_la_SOURCES = \
 			   odp_pkt_queue.c \
 			   odp_queue_if.c \
 			   queue/subsystem.c \
-			   queue/generic.c \
-			   queue/scalable.c \
 			   odp_rwlock.c \
 			   odp_rwlock_recursive.c \
 			   odp_schedule_if.c \
-			   schedule/generic.c \
-			   schedule/iquery.c \
-			   schedule/scalable.c \
-			   schedule/scalable_ordered.c \
-			   schedule/sp.c \
```

Comment:
Again it's not obvious how these are buffer changes.

Reply:
Agree. The makefile changes should be a separate commit as well.